kb/temporal-platform-limits-sheet.md

```diff
@@ -9,38 +9,38 @@ date: 2023-04-28T00:00:00Z
 ---
 
 Running into limits can cause unexpected failures.
-Knowing Temporal's limits can prevent that.
+Knowing the limits of Temporal can prevent that.
 
-This page details many of the hard and soft limits that are coded into the Temporal Platform.
-Hard limits fail with an error; soft limits produce a warning log on the server side.
+This page details many of the hard and soft limits coded into the Temporal Platform.
+Hard limits fail with an error. Soft limits produce a warning log on the server side.
 
 <!-- truncate -->
 
-- **Identifiers**: By default, the maximum length for identifiers (like Workflow Id, Workflow Type, and Task Queue name) is 1000 characters.
-  - This is configurable with the `limit.maxIDLength` dynamic config, which is set to 255 in [our SQL example](https://github.com/temporalio/docker-compose/blob/93d382ef9133e4cde8ce311de5153cd0cc9fbd0c/dynamicconfig/development-sql.yaml#L1-L2).
+- **Identifiers**: By default, the maximum length for identifiers (such as Workflow Id, Workflow Type, and Task Queue name) is 1000 characters.
+  - This is configurable with the `limit.maxIDLength` dynamic config variable, set to 255 in [this SQL example](https://github.com/temporalio/docker-compose/blob/93d382ef9133e4cde8ce311de5153cd0cc9fbd0c/dynamicconfig/development-sql.yaml#L1-L2).
   - The character format is UTF-8.
 - **gRPC**: gRPC has a limit of 4 MB for [each message received](https://github.com/grpc/grpc/blob/v1.36.2/include/grpc/impl/codegen/grpc_types.h#L466).
 - **Event batch size**: The `DefaultTransactionSizeLimit` limit is [4 MB](https://github.com/temporalio/temporal/pull/1363).
-  This is the largest transaction size we allow for Event Histories to be persisted.
+  This is the largest transaction size allowed for the persistence of Event Histories.
 - **Blob size limit** for Payloads (including Workflow context and each Workflow and Activity argument and return value; _[source](https://github.com/temporalio/temporal/blob/v1.7.0/service/frontend/service.go#L133-L134)_):
-  - We warn at 256 KB: [Blob size exceeds limit.](https://github.com/temporalio/temporal/blob/fee1c43823699e90b330680a8efeb9d8dbee8cf3/common/util.go#L568)
-  - We error at 2 MB: `ErrBlobSizeExceedsLimit: Blob data size exceeds limit.`
+  - Temporal warns at 256 KB: [Blob size exceeds limit.](https://github.com/temporalio/temporal/blob/fee1c43823699e90b330680a8efeb9d8dbee8cf3/common/util.go#L568)
+  - Temporal errors at 2 MB: `ErrBlobSizeExceedsLimit: Blob data size exceeds limit.`
```

Comment:
I'm not sure is error logs are useful for customers which do not have access to service logs?

Reply:
Trying to avoid saying "we". This article is for people that would have access to the logs